gisutils/vector.py

```diff
@@ -207,3 +208,79 @@ def glue_lines_together(gdf, seed_id, id_col, geom_col='geometry', max_tries=Non
             success += 1
 
     return line, extra_geoms
+
+
+def line_to_point(line_geom,interval_dist):
+    """
+    Create a series of points x distance apart along a line geometry
+    """
+    points = [line_geom.interpolate(d) for d in numpy.arange(0, line_geom.length, interval_dist)]
+    return points
+
+def compute_ortho_bearing(bearings):
+    """
+    Create orthogonal bearing from the bearings of points on a polyline
+    """
+    ortho_bearing = (bearings - numpy.pi*0.5)%(2*numpy.pi)
+    return ortho_bearing
+
+def draw_ortho_line_to_bank(ct_pt_geom, bank_geom, ortho_bearing, step, buffer_dist, max_dist):
+    """ 
+    Search for intersects between the orthogonal lines and the river banks until max_dist is reached.
```

Comment:
docstrings should be in the numpy doc format: https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt